spikes/npgsql-vanilla/README.rst

```diff
@@ -0,0 +1,40 @@
+#################################################################
```

Comment:
we should probably use the standard syntax for titles and headers (here and throughout the rest of the file)

see https://github.com/crate/crate-docs/blob/main/style/rst.rst#titles-and-headings

Reply:
Thank you. Addressed with 5960fa3.